integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala

```diff
@@ -320,13 +322,8 @@ class AlterTableColumnSchemaGenerator(
 // TODO: move this to carbon store API
 object TableNewProcessor {
   def apply(
-      cm: TableModel,
-      identifier: AbsoluteTableIdentifier): TableInfo = {
-    new TableNewProcessor(
-      cm,
-      identifier.getDatabaseName,
-      identifier.getTableName,
-      identifier.getTablePath).process
+      cm: TableModel): TableInfo = {
```

Comment:
Do not change this, otherwise tablePath will be lost

Reply:
We have to pass the TableInfo to the CarbonCreateTableCommand so that we can use the 
CarbonCreateTableCommand  flow from the RefreshCarbonTableCommand.
Any ways for supporting the external table path, the tablePath is being passed to CarbonCreateTableCommand